draft-ietf-jsonpath-base.md

```diff
@@ -1646,6 +1647,23 @@ exponential, CPU usage or, for example via a naive recursive implementation of t
 stack overflow. Implementations need to have appropriate resource management
 to mitigate these attacks.
 
+## Attack Vectors on How JSONPath Queries are Formed
+
+JSONPath queries are often not static, but formed from variables that
+provide index values, member names, or values to compare with in a
+filter expression.
+These variables need to be translated into the form they take in a
+JSONPath query, e.g., by escaping string delimiters, or by only
+allowing specific constructs such as `.name` to be formed when the
+given values allow that.
+Failure to perform these translations correctly can lead to unexpected
+failures, which can lead to Availability, Confidentiality, and
+Integrity breaches, in particular if an adversary has control over the
+values (e.g., by entering them into a Web form).
+The resulting class of attacks, *injections* (e.g., SQL injections),
+is consistently found among the top causes of application security
+vulnerabilities and requires particular attention.
```

Comment:
Would be improved by examples?

Reply:
I am reluctant here. Examples might be used as starting point for potential attackers.

Reply:
@goessner I understand your reluctance, but the examples are likely to be simple attacks which anyone with the relevant background (attackers, penetration testers, etc.) would think of immediately.